QuickGuidePanels/index.js

```diff
@@ -15,19 +15,19 @@
  */
```

Comment:
Please also fix `QuickGuidePanel` at line 13.

Reply:
It's fixed.